CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.3.1
```

Comment:
This should say "Next release" since we don't know what the version number will be yet